experimental/generation/generator/packages/library/test/forms/unittest_percentage.schema

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "Percentage": {
+            "type": "string",
+            "$entities": [
+                "percentage"
+            ]
+        }
+    },
+    "required": [
+        "Percentage"
+    ],
+    "$requires": [
+        "standard.schema"
+    ]
```

Comment:
Technically speaking this is the default and we should probably remove these everywhere except when you want to add chit-chat.